pkgs/by-name/pi/pingtunnel/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "pingtunnel";
+  version = "2.8";
+
+  src = fetchFromGitHub {
+    owner = "esrrhs";
+    repo = "pingtunnel";
+    rev = version;
+    hash = "sha256-qkIeE6gNBUUd5/IhcKhDmXZbzeSTR0r6qCAK/Nz7Kew=";
+  };
+
+  vendorHash = "sha256-SkBHFvQn3Q6fl/d1npEspTYhq1YrnusXNWr/Ky4bv64=";
+
+  ldflags = [ "-s" "-w" ];
+
+  postInstall = "mv $out/bin/cmd $out/bin/pingtunnel";
```

Comment:
Have you tried https://nixos.org/manual/nixpkgs/unstable/#var-go-subPackages?

Reply:
> Have you tried https://nixos.org/manual/nixpkgs/unstable/#var-go-subPackages?

Tried adding `subPackages = ["cmd"];` but makes no difference. Result still produce `$out/bin/cmd`, needs manual mv.

Reply:
Have you tried `subPackages = ["pingtunnel"];`?

Reply:
> Have you tried `subPackages = ["pingtunnel"];`?

This will report `stat /build/source/pingtunnel: directory not found`